jdaviz/core/helpers.py

```diff
@@ -315,6 +315,32 @@ def show(self, loc="inline", title=None):
         as only JupyterLab has a mechanism to have multiple tabs.
         """
         try:
+            # Check if the user is running Jdaviz in the correct environments.
+            # If not, provide a friendly msg to guide them!
+            try:
+                shell = get_ipython().__class__.__name__  # noqa
+                if shell == 'TerminalInteractiveShell':
+                    raise RuntimeError("IPython shell not supported")
+                elif shell != 'ZMQInteractiveShell':
+                    raise RuntimeError("Unknown shell not supported")
+            except Exception as orig_e:
+                if type(orig_e) is NameError:
+                    orig_e = RuntimeError("Standard Python Interpreter not supported")
+
+                boilerplate_e = \
+                    RuntimeError('\nYou are currently running Jdaviz from an unsupported '
+                                 'shell. Jdaviz is intended to be run within a Jupyter '
+                                 'notebook, or directly from the command line.\n\nTo run '
+                                 'from Jupyter, call <your viz>.show() from a notebook cell.\n'
+                                 'To run from the command line, run '
+                                 'jdaviz --layout=<your viz> outside of Python.\n\n'
+                                 'To learn more, see "Importing Data" from your config docs at: '
+                                 'https://jdaviz.readthedocs.io/en/latest/index.html\n'
+                                 'For some working examples, see our sample notebooks at: '
+                                 'https://jdaviz.readthedocs.io/en/latest/sample_notebooks.html\n\n'
+                                 'Thanks for trying out Jdaviz! :)')
```

Comment:
I'm not sure this part is necessary, but if you want to leave it in, we should probably format to point to the actual version tag instead of always to latest.  And we'll need to make sure if we ever move `sample_notebooks.html`, that we update this too.